Example/iOS-Example/SwiftUIView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  SwiftUIView.swift
+//  iOS Example
+//
+//  Created by Ansel Merino on 5/31/23.
+//  Copyright © 2023 Speramus Inc. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+import FetchRequests
+
+struct SwiftUIView: View {
+    @FetchableRequest(
+        definition: Model.fetchDefinition(),
+        sortDescriptors: [
+            NSSortDescriptor(
+                key: #keyPath(Model.updatedAt),
+                ascending: false
+            ),
+        ],
+        animation: Animation.easeIn(duration: 1.0)
+    )
+    
+    private var models: FetchableResults<Model>
+
+    var body: some View {
+        NavigationView {
+            List(models) { model in
+                row(for: model)
+            }
+            .listStyle(PlainListStyle())
+            .transition(.slide)
+            .navigationBarTitle("SwiftUI Example", displayMode: .inline)
+            .navigationBarItems(
+                leading:
+                    Button {
+                        Model.reset()
+                    } label: {
+                        Image(systemName: "trash")
+                    },
+                trailing:
+                    Button {
+                        try? Model().save()
+                    } label: {
+                        Image(systemName: "plus")
+                    }
+            )
+        }
+    }
+    
+    static var viewController: UIHostingController<SwiftUIView> {
```

Comment:
Interesting... haven't seen this pattern before...
But will ask that you move this into UIKit and keep the view "pure"